renderer/src/pages/dashboard/Dashboard.tsx

```diff
@@ -4,6 +4,7 @@ import ChartController from './ChartController'
 import useStationRewards from 'src/hooks/StationRewards'
 import RewardsInfo from './RewardsInfo'
 import { bigIntFilToNumber } from 'src/lib/utils'
+import type { JSX } from 'react'
```

Comment:
JSX is now part of the react module and (apparently) not exposed as a global any more